conans/client/cache/remote_registry.py

```diff
@@ -181,19 +214,25 @@ def get(self, remote_name):
 
     def __getitem__(self, remote_name):
```

Comment:
Where is this getter called and which operation is blocking if the remote is disabled?

Reply:
The calls to this getter come from commands in which you can specify the remote
So it raises an exception in case you want to do:
`conan install pkg -r disabled_remote`